Makefile

```diff
@@ -209,9 +209,10 @@ build-ecs-cni-plugins:
 
 build-vpc-cni-plugins:
 	@docker build --build-arg GOARCH=$(GOARCH) --build-arg GO_VERSION=$(GO_VERSION) -f $(VPC_CNI_REPOSITORY_DOCKER_FILE) -t "amazon/amazon-ecs-build-vpc-cni-plugins:make" .
-	docker run --rm --net=none \
-		-e GO111MODULE=off \
+	docker run --rm --net=host \
```

Comment:
We need network to download the dependencies. 

According to https://go.dev/ref/mod#mod-commands

> In module-aware mode, the go command uses go.mod files to find versioned dependencies, and it typically loads packages out of the [module cache](https://go.dev/ref/mod#glos-module-cache), downloading modules if they are missing. 

We used to use vendor directory before that's why it does not require network. 

> In GOPATH mode, the go command ignores modules; it looks in [vendor directories](https://go.dev/ref/mod#glos-vendor-directory) and in GOPATH to find dependencies.

I'm making the change because the source amazon-vpc-cni-plugins [repo](https://github.com/aws/amazon-vpc-cni-plugins) also builds in module-aware mode. Although it has a vendor directory, the dependency version has been out-of-sync. 

If we believe the need for internet during a build should be avoided as much as possible, we can go back to using vendor directory by asking the CNI plugin maintainers to keep the vendor directory up to date.

Reply:
I would suggest we go and fix CNI plugin to keep the vendor directory up to date.

Reply:
Chatted with @ofiliz, maintainer of `amazon-vpc-cni-plugins` repo - Onur is going to handpick all necessary dependencies because running `go mod vendor` brings a much larger dependency set than what we actually need and is undesirable.

Reply:
https://github.com/aws/amazon-vpc-cni-plugins/pull/73